CHANGELOG.md

```diff
@@ -19,7 +19,8 @@
 
 #### Bug Fixes:
 * Custom content no longer clobbers `col1_header` table configs
-
+* The option `--file-list` that refers to a text file with file paths to analyse will no longer ignore directory paths
```

Comment:
I'd say that this is a new feature! 😉

Reply:
In other words ```It's not a bug, it's a feature!```  ;-)

I doubted because I originally expected the `--file-list` to already do what this PR entails.